# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

moveToActionGoal action_goal
moveToActionResult action_result
moveToActionFeedback action_feedback
